pytorch_lightning/loggers/neptune.py

```diff
@@ -342,3 +397,35 @@ def version(self) -> str:
         except NeptuneOfflineModeFetchException:
             return 'offline-id-1234'
         return self.run['sys/id'].fetch()
+
+    def _raise_deprecated_api_usage(self, f_name, sample_code):
+        raise ValueError(f"Function you've used  is deprecated."
```

Comment:
Change to:
The function you've used is deprecated.
If you are looking for the Neptune logger using legacy Python API it has been renamed to NeptuneLegacyLogger. The NeptuneLogger was re-written to use the neptune.new Python API (learn more: https://neptune.ai/blog/neptune-new).
Instead of `logger.{f_name}` you can use:
\t{sample_code}

Reply:
ok